redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: crwoperator.v1.1.0
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion":"org.eclipse.che/v1",
+          "kind":"CheCluster",
```

Comment:
Just noticed this; this API is also provided by the EclipseChe operator: https://github.com/operator-framework/community-operators/pull/280/files#diff-e2003439d400d84637f240ca92eeb43bR158
If an admin installs both operators, this could be an issue
@ecordell

Reply:
Yes, when I tried to put 2 packages to quay - Che and CRW - I had reconciliation errors in marketplace operator logs.

Reply:
@eivantsov That doesn't sound good. I'm not sure what this operator does versus what the eclipse che operator does, so I'm not sure I understand why they use the same CR.

Reply:
@eivantsov Nevermind, I think I see now that this is the downstream Red Hat version, and Eclipse Che is the community version. Nevermind my previous comment

Reply:
@SamiSousa operators are identical, though CRW operator deploys CRW, Eclipse Che - upstream server image. There's a field in CR spec that instructs the operator on what flavor of Eclipse Che to use - upstream vs RH product. For example, CRW operator will deploy postgres from RHCC, RS SSO and CRW server image. Eclipse Che operator will deploy postgres from DocjerHub, Keycloak and Eclipse Che server image. The idea is to keep one codebase and one set of APIs.

Reply:
@SamiSousa @ecordell will two operators coming from different operator sources but requesting the same CRD work ok?

Please, note that single mode install is what they use, ie operators will watch only their namespaces.